### ISSUE TITLE
Component [app] no longer exists.

### ISSUE THREAD

**Issue by gaaclarke** [dart-lang/appengine#130]
The documentation for the appengine package mentions running `gcloud components update app` but the warning `WARNING: Component [app] no longer exists.` will appear in the following version:
```
Google Cloud SDK 280.0.0
bq 2.0.53
core 2020.02.07
gsutil 4.47
```

**Comment by XilaiZhang** [comment]
Yeah the app component automatically exists after downloading the gcloud sdk. (https://stackoverflow.com/questions/43244074/cant-access-publicly-exposed-docker-container-port-from-external-machine-only) would be great if author of this repo update the README.md and remove the `gcloud components update app` line
